dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -86,23 +87,26 @@ private static Repository createClassRepository(List<Path> paths) {
   static ClassDumper create(List<Path> jarPaths) throws IOException {
     ClassLoader systemClassLoader = ClassLoader.getSystemClassLoader();
     ClassLoader extensionClassLoader = systemClassLoader.getParent();
+    jarPaths.forEach(
+        jar -> {
```

Comment:
Looks like this PR needs to be updated to current master.

Reply:
Merged.